### DISCUSSION TITLE
next js + react relay => ssr => big big data into script tag inside html body

### DISCUSSION THREAD

**Discussion by vunguyenhieu001** [vercel/next.js#38378] [Help]
My apologies if this topic was came up, somehow
I need your help, here is my problem.
First, the context, my page useingssr mode
getServerSideProps request to react-relay server, then i send the queryRecords on client side
using inside to init relayenvironment
<RelayEnvironmentProvider environment={initEnvironment({ records: queryRecords })}>
then inside some component i use react-relay/usequery
useQuery(query, {}, { fetchPolicy: 'store-only' });
that request get data from RelayEnvironmentProvider
and i have my data everything is good

**BUT**

when i look into the html source code, i have a big big data inside this
<script id="__NEXT_DATA__" type="application/json">

**my question**
when my page is generating in build prod mode, how could i do to clean this data 

<script id="__NEXT_DATA__" type="application/json"> 

**===================**

Thanks for your help
by the way, it's my really first topic on github, be kind please, thanks a lot

**Comment by thomasleblanc** [comment]
up!
